clidocstool_md.go

```diff
@@ -227,3 +228,7 @@ func mdCmdOutput(cmd *cobra.Command, old string) (string, error) {
 func mdEscapePipe(s string) string {
 	return strings.ReplaceAll(s, `|`, `\|`)
 }
+
+func mdReplaceNewline(s string) string {
+	return regexp.MustCompile(`\r\n?|\n`).ReplaceAllString(s, "<br>")
```

Comment:
Better to put the `MustCompile` either in a `sync.Once` or as a package `var`.

The regex probably should be making the `\r` optional, and not the `\n` (`\r` newlines are AFAIK only occurring on classic macOS);


```suggestion
	return regexp.MustCompile(`\r?\n`).ReplaceAllString(s, "<br>")
```

Reply:
Oh; is XHTML still a thing nowadays? (I still have the tendency to use `<br />`, but perhaps we don't really care about that)

Reply:
> Oh; is XHTML still a thing nowadays? (I still have the tendency to use `<br />`, but perhaps we don't really care about that)

`<br>` is enough imo as markdown content is not served as `application/xhtml+xml` anyway (considered [harmful](http://hixie.ch/advocacy/xhtml)). Also with HTML5 even if you write `<br/>` or `<br />` it will eventually be converted to `<br>` by the browser.

Reply:
> considered [harmful](http://hixie.ch/advocacy/xhtml)

Heh, yes, a lot of that was written when Internet Explorer was still "the largest browser", and, well, a lot of fun was needed in either case to write HTML that worked both in IE, and in browsers that were _actually_ following standards 😂 

But yeah, we don't do anything with XHTML(5), so 😂